src/figuresClasses.ts

```diff
@@ -1,11 +1,96 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  private readonly a: number;
+
+  private readonly b: number;
+
+  private readonly c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side lengths be greater than 0');
```

Comment:
There is a typo in the error message: 'Side lengths be greater than 0'. It should be 'Side lengths must be greater than 0'.